pkg/remotewrite/remotewrite.go

```diff
@@ -245,7 +252,7 @@ func (o *Output) convertToPbSeries(samplesContainers []metrics.SampleContainer)
 				swm.Latest = truncTime
 				o.tsdb[sample.TimeSeries] = swm
 				seen[sample.TimeSeries] = struct{}{}
-			} else {
+			} else { //nolint:gocritic
```

Comment:
What is the error here? :thinking:

Reply:
It complains that the `else { if {` could be replaced with `else if {`. This won't be true anymore, once the TODOs are addressed. Thus I deactivated it locally.

Reply:
Can you add it as a comment to the nolint, please? I will look into it